axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -358,6 +362,14 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    public boolean isShortcutQueriesToLocalHandlers() {
+        return localSegmentShortCut;
+    }
+
+    public void setLocalSegmentShortCut(boolean localSegmentShortCut) {
```

Comment:
I think we should rename the setter in a similar fashion as the getter.
So, to something like this:
```suggestion
    public void setShortcutQueriesToLocalHandlers(boolean shortcutQueriesToLocalHandlers) {
```